### PR TITLE
.NET Text Analytics July release notes

### PR DESCRIPTION
**Pull request by maririos**

### CODE REVIEW

File: releases/2020-07/dotnet.md [hunk 1]
```diff
@@ -10,35 +10,22 @@ The Azure SDK team is pleased to announce our July 2020 client library releases.
...
 
 #### Preview
 
-- Management Library - Compute
-- Management Library - Network
-- Management Library - Resources
-- Management Library - Storage
-- Management Library - KeyVault
-- Management Library - EventHubs
-- Management Library - AppConfiguration
```
Comment (ShivangiReja) [inline releases/2020-07/dotnet.md:27]: We don't want to remove management libraries. Maybe we can add two different headings to differentiate both?

Comment (maririos) [reply releases/2020-07/dotnet.md]: ups. Added back

File: releases/2020-07/dotnet.md [hunk 2]
```diff
@@ -51,10 +52,17 @@ If you have a bug or feature request for one of the libraries, please [file an i
...
+A new set of management libraries that follow the [Azure SDK Design Guidelines for .NET](https://azure.github.io/azure-sdk/dotnet_introduction.html) and based on [Azure.Core libraries](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/core/Azure.Core) are now in Public Preview. These new libraries provide a number of core capabilities that are shared amongst all Azure SDKs, including the intuitive Azure Identity library, an HTTP Pipeline with custom policies, error-handling, distributed tracing, and much more. You can find the list of new packages [on this page](https://azure.github.io/azure-sdk/releases/latest/dotnet.html).
 
 To get started with these new libraries, please see the [quickstart guide here](https://github.com/Azure/azure-sdk-for-net/blob/master/doc/mgmt_preview_quickstart.md). These new libraries can be identifed by namespaces that start with `Azure.ResourceManager`, e.g. `Azure.ResourceManager.Network`
 
+### Text Analytics 
+
+#### 1.0.1 [Changelog](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md#101-2020-06-23)
```
Comment (ShivangiReja) [inline releases/2020-07/dotnet.md:61]: Can we merge these two lines like this - https://github.com/Azure/azure-sdk/blob/master/releases/2020-06/dotnet.md#form-recognizer-changelog?

Comment (maririos) [reply releases/2020-07/dotnet.md]: updated

**Pull request merged**